app-office/Focuswriter/focuswriter-1.6.7.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="A QT Word Processor"
+DESCRIPTION="Focuswriter is an Open Source QT Word Processor"
+HOMEPAGE="https://gottcode.org/focuswriter/"
+COPYRIGHT="2008-2017 Graeme Gott"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/portableapps/files/Source/FocusWriter/focuswriter-1.6.7-src.tar.bz2"
```

Comment:
Replace the version number from focuswriter-1.6.7-src.tar.bz2 to focuswriter-$portVersion-src.tar.bz2.